src/main/java/org/trigon/hardware/simulation/SimpleMotorSimulation.java

```diff
@@ -1,41 +1,75 @@
 package org.trigon.hardware.simulation;
 
 import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.system.plant.LinearSystemId;
 import edu.wpi.first.math.util.Units;
 import edu.wpi.first.wpilibj.simulation.DCMotorSim;
 import org.trigon.hardware.RobotHardwareStats;
 
-
+/**
+ * A class that represents a simulation of a simple motor mechanism.
+ */
 public class SimpleMotorSimulation extends MotorPhysicsSimulation {
     private final DCMotorSim motorSimulation;
 
-    public SimpleMotorSimulation(DCMotor gearbox, double gearRatio, double momentOfInertia) {
+    /**
+     * Creates a new SimpleMotorSimulation.
+     *
+     * @param gearbox   The gearbox of the motor(s)
+     * @param gearRatio The gearbox's gear ratio
+     * @param kv        voltage needed to maintain constant velocity
+     * @param ka        voltage needed to induce a specific acceleration
+     */
+    public SimpleMotorSimulation(DCMotor gearbox, double gearRatio, double kv, double ka) {
```

Comment:
Make another constructor for inputting moment of inertia.

Reply:
DCMotorSim doens't have a constructor with moi. It only has one with ka and kv